src/qibocal/protocols/characterization/utils.py

```diff
@@ -259,6 +266,55 @@ def fit_punchout(data: Data, fit_type: str):
     return [low_freqs, high_freqs, ro_values]
 
 
+def fill_table(
+    qubit: QubitId,
+    name: str,
+    value: float,
+    error: Optional[float],
+    unit: str = None,
+    ndigits: int = 2,
+) -> str:
+    """
+    Return a row of the report table with the correct number of
+    significant digits.
+
+    Args:
+        qubit (QubitId): Qubit.
+        name (str): Variable's name.
+        value (float): Variable's value.
+        error (float): Error associated to the variable.
+        unit (str): Measurement unit. Default value `None`.
+        ndigits (int): Number of decimal digits to display when error is `None`
+            (i.e. it is not evaluated).
+    """
+    table = f"{qubit}| {name}: "
+    if value:
+        magnitude = floor(log10(abs(value)))  # number of non decimal digits in value
+    else:
+        magnitude = 0
+    if error:
+        ndigits = max(significant_digit(error * 10 ** (-1 * magnitude)), 0)
+        table += f"({round(value*10**(-1*magnitude), ndigits)} {chr(177)} {np.format_float_positional(round(error*10**(-1*magnitude), ndigits), trim = '-')})"
```

Comment:
Python supports Unicode (i.e. even in its source), if you want to use `±` just do:
```suggestion
        table += f"({round(value*10**(-1*magnitude), ndigits)} ± {np.format_float_positional(round(error*10**(-1*magnitude), ndigits), trim = '-')})"
```